packages/jest-matcher-utils/src/index.js

```diff
@@ -13,8 +13,22 @@
 const chalk = require('chalk');
 const prettyFormat = require('pretty-format');
 const AsymmetricMatcherPlugin = require('pretty-format/build/plugins/AsymmetricMatcher');
-
-const PLUGINS = [AsymmetricMatcherPlugin];
+const ImmutableOrderedSetPlugin = require('pretty-format/build/plugins/ImmutableOrderedSet');
+const ImmutableListPlugin = require('pretty-format/build/plugins/ImmutableList');
+const ImmutableMapPlugin = require('pretty-format/build/plugins/ImmutableMap');
+const ImmutableOrderedMapPlugin = require('pretty-format/build/plugins/ImmutableOrderedMap');
+const ImmutableSetPlugin = require('pretty-format/build/plugins/ImmutableSet');
+const ImmutableStackPlugin = require('pretty-format/build/plugins/ImmutableStack');
+
+const PLUGINS = [
+  AsymmetricMatcherPlugin, 
+  ImmutableOrderedSetPlugin, 
+  ImmutableListPlugin,
+  ImmutableOrderedMapPlugin,
+  ImmutableMapPlugin,
+  ImmutableSetPlugin,
+  ImmutableStackPlugin,
+];
```

Comment:
We also use plugins in `jest-diff` and `jest-snapshot`.
Alos, maybe we could create `ImmutablePlugins.js` which exposes an array of all Immutable plugins?